pkgs/development/libraries/aspell/dictionaries.nix

```diff
@@ -713,6 +788,12 @@ in rec {
     fullName = "Polish";
     filename = "aspell6";
     sha256 = "0kap4kh6bqbb22ypja1m5z3krc06vv4n0hakiiqmv20anzy42xq1";
+    meta.license = with lib.licenses; [
+      gpl2Only
+      lgpl21Only
+      mpl11
+      cc-by-10 # not exactly but close enough
```

Comment:
Well the Copyright says It's CC-SA-1.0, but in any case CC-BY-SA-1.0 is closer than CC-SA-1.0.
Since CC-BY-SA-1.0 is not known to SPDX, CC-SA-1.0 is what the license checker tool told me.

Reply:
Should be added to lib/license.nix

Reply:
Sorry miss read the license, did not even realize CC-SA-1.0 should be added to lib/license.nix file.